jmeter-java-dsl-graphql/src/test/java/us/abstracta/jmeter/javadsl/graphql/DslGraphqlSamplerTest.java

```diff
@@ -137,6 +138,41 @@ public void shouldSendGraphqlQueryToServerWhenGraphqlSamplerWithHttpGet()
     ).run();
     verify(getRequestedFor(anyUrl()).withQueryParam("query", equalTo(QUERY)));
   }
+  @Test
+  public void shouldThrowExceptionForInvalidVariable() {
+    DslGraphqlSampler sampler = graphqlSampler("http://localhost", "{ user(id: 1){ name } }");
+    assertThrows(IllegalArgumentException.class, () -> sampler.variable("invalidVar", new Object()));
+  }
+
+  @Test
+  public void shouldHandleLongOperationName() throws Exception {
+    String longOperationName = "a".repeat(300);
+    testPlan(
+            threadGroup(1, 1,
+                    graphqlSampler(wiremockUri, QUERY)
+                            .operationName(longOperationName)
+            )
+    ).run();
+    verify(postRequestedFor(anyUrl())
+            .withRequestBody(equalToJson(buildRequestBody(longOperationName, null))));
+  }
+  @Test
```

Comment:
What's the difference between this test and the one with Get? I don't see any "put" in test body.